### ISSUE TITLE
I really like this call-to-action on Columbus landing page

### ISSUE THREAD

**Issue by jonfroehlich** [ProjectSidewalk/SidewalkWebpage#2340]
I quite like the CTA on the Columbus landing page: "Live in Columbus? Stop by a mapathon event in your area!" I think we could adapt this to other landing pages to provide semi-automatically generated CTAs (like: "Just need 2 miles more in [neighborhood], click here to start")

![image](https://user-images.githubusercontent.com/1621749/97199336-22580500-176d-11eb-856c-fd4e096f0de9.png)

**Comment by misaugstad** [comment]
We should also see if we can get a link like this for SPGG and CDMX that always points to the next mapathon!

**Comment by jonfroehlich** [comment]
Right, good idea! I was also thinking it would be good to have CTAs to help us meet the next goal (e.g., finish a neighborhood) that we could show when there are no planned mapathons.

**Comment by misaugstad** [comment]
Yeah I think it's easy enough to just start with finding the neighborhood with the fewest miles remaining to finish it, and just using the sentence you wrote up there.

And @gari01234 would it be possible to create a permanent URL that always points to an event page for the next mapathons in SPGG and CDMX?
